new/validateform.js

```diff
@@ -0,0 +1,27 @@
+const form = document.getElementById('fs-frm');
+const email = document.getElementById('useremail');
+const errorUpper = '&#9888; Your email is in uppercase. Please rewrite it in lowercase.';
+const small = document.createElement('small');
```

Comment:
- [ ] Kindly make sure the error message is showed up close to the **Submit* button as required for the project. Right now it is far from the **Submit* button and if the user doesn't scroll the whole page, he cannot see the **error message.** I suggest you display between the `textearea` field and the `Get In Touch` button.